pages/entry.jsx

```diff
@@ -66,22 +66,28 @@ export default React.createClass({
   },
   render() {
     let docTitle;
+    let description;
     let justPostedByUserMessage = null;
     let content;
 
     if (this.state.dataLoaded) {
       docTitle = `${this.state.entry.title}`;
       justPostedByUserMessage = this.state.justPostedByUser ? (<h5 className="col-12 text-center">Thanks for submitting!</h5>) : null;
       content = <ProjectCard {...Utility.processEntryData(this.state.entry)} onDetailView={true} />;
+      description = `${this.state.entry.description}`;
     }
 
     if (this.state.noData) {
       docTitle = NO_ENTRY_TITLE;
       content = NO_ENTRY_BLOCK;
+      description = '';
```

Comment:
we probably want to declare a new `NO_ENTRY_OG_DESCRIPTION` that has a generating entry string so that when search engines/twitter/etc try to read in that value, it has something in there, rather than being an empty string.